src/newrelic/processor.go

```diff
@@ -365,10 +365,15 @@ func (p *Processor) processAppInfo(m AppInfoMessage) {
 		return
 	}
 
-	if len(p.apps) > limits.AppLimit {
+	numapps := len(p.apps)
+	log.Debugf("number of apps is %d", numapps)
```

Comment:
This would generate a LOT of debug messages.  It doesn't seem necessary with the high low messages already being generated. Same comment for https://github.com/newrelic/newrelic-php-agent/pull/534/files#diff-d34a767f316aaeded794ac3a231f867d3ffb42de19365ed338eace17f5876d94R716

Reply:
It only occurs if a new app connects to the daemon.  Do customers have lots of new apps connecting constantly?

Reply:
Removed in commit [182a501](https://github.com/newrelic/newrelic-php-agent/pull/534/commits/182a501d2d825404652e790fbefb8042128208ec).